### ISSUE TITLE
Edit property link redundant on properties  screen

### ISSUE THREAD

**Issue by kevinandrews1** [bbertucc/equalify#359]
The “Edit Property” link lacks context and is redundant, especially for users navigating via the tab key or using a links list, which is common with modern screen readers. To improve accessibility, change the link text to “Edit Accessibility 1 Property” or use the specific property name. Ensure that the accessible name and visible label match to provide a consistent experience. This adjustment will benefit speech-input users, allowing them to activate controls directly with fewer focus changes, and text-to-speech users will have an improved experience because the labels they hear will match the visible text labels on the screen.

**Comment by bbertucc** [comment]
@wilsuriel03 you want to look at this? You can visually hide the property name from the button, so it only reads out the full text for screen readers, but keeps the visual button saying "Edit Property"

**Comment by wilsuriel03** [comment]
Complete ✅
